outscale/resource_outscale_volume_link_test.go

```diff
@@ -53,7 +53,7 @@ func TestAccVM_ImportVolumeAttachment_Basic(t *testing.T) {
 		CheckDestroy: testAccCheckOAPIVolumeAttachmentDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccOAPIVolumeAttachmentConfig(omi, "tinav4.c2r2p2", utils.GetRegion(), keypair),
```

Comment:
Stupid question, but as you use the same value in all vm, wouldn't it be better to store it in a global, or function so you can change it only in 1 file ?